app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -116,13 +124,24 @@ public function verifyOrderContents($observer)
                 Mage::helper('boltpay/bugsnag')->notifyException($e, $metaData);
             }
 
-            $order->sendNewOrderEmail()
-                ->addStatusHistoryComment('Email sent for order ' . $order->getIncrementId())
-                ->setIsCustomerNotified(true)
-                ->save();
+            $this->sendOrderEmail($order);
         }
     }
 
+    public function proceedTransmit($complete_authorize_request)
```

Comment:
naming is a bit unclear; I think it should be something like `sendCompleteAuthorizeRequest`